release_notes/48-release-notes.adoc

```diff
@@ -42,7 +42,33 @@ Policy::
 
 This release adds improvements related to the following components and concepts:
 
+//ROX-20769
+[id="central-db-postgresql_15_{context}"]
+=== Central DB uses PostgreSQL 15
 
+Central DB component is using PostgreSQL 15 now, and {product-title-short} 4.8
+supports this version for external databases as well. A new installation with
+an internal database will use this version by default now. When upgrading an
+existing cluster to {product-title-short} 4.8, Central DB will do an in-place
+upgrade of the data collected so far.
+
+[NOTE]
+====
+To make the database upgrade process smooth, we recommend:
+
+* Take a backup before upgrading to {product-title-short} 4.8
+
+* If not using the Operator: check disk space available for the database
+`rox_central_postgres_available_size_bytes`. For the purposes of the upgrade,
```

Comment:
```suggestion
`rox_central_postgres_available_size_bytes` variable. For the purposes of the upgrade,
```
Not sure if "variable" or "parameter" is correct here, but I think "variable"?

Reply:
Probably "metric" would be better.